### ISSUE TITLE
Chaining everything

### ISSUE THREAD

**Issue by bongobongoland** [mpgn/CrackMapExec#123]
### Discussed in https://github.com/mpgn/CrackMapExec/discussions/117

^^^ in that particular case, chaining -x & -M lsassy.
However.
I think chaining everything would be beneficial :) 
For instance -x & --sam

**Comment by XiaoliChan** [comment]
@bongobongoland This is what you want?
![image](https://github.com/mpgn/CrackMapExec/assets/30458572/519851c0-f332-46ce-a62d-a890ab419385)

**Comment by bongobongoland** [comment]
Yes. Something like that. Do we really need --chain?

**Comment by XiaoliChan** [comment]
> Yes. Something like that. Do we really need --chain?

I have no idea...

**Comment by bongobongoland** [comment]
Probably not :) 
I think the order of arguments itself is descriptive enough.
If I type -x whoami and then -M enum_av it should mean just that: run the command first, then run the module.
I could be wrong though. Since currently the modules can only be chained using -M between them.

**Comment by mpgn** [comment]
It's already there  normaly

https://twitter.com/mpgn_x64/status/1679085727785574402?t=f4CWS_rNq0ql5gqAnSVWPA&s=19

**Comment by bongobongoland** [comment]
I didn't know that. Then I guess we are talking about the ability to add -x to the chain.

**Comment by mpgn** [comment]
I will think about it, not sure it's a good idea to mess -x with other options :)

**Comment by bongobongoland** [comment]
the ability to add -x would be useful in some other cases, for instance when running the rdp module

**Comment by bongobongoland** [comment]
Here's another example. issues/125
Disabling firewall with -x and then running the rdp module :)

**Comment by XiaoliChan** [comment]
> Here's another example. issues/125
> Disabling firewall with -x and then running the rdp module :)

nope, if you check the rdp module source code, you can find I have write two method, the wmi method is for quick configure rdp, the Smb method will doing execute command

**Comment by XiaoliChan** [comment]
My innovation is, when the wmi (doom) is blocked, users can switch to smb
